### PR TITLE
add test coverage for AllCasaAdmins::CasaAdminsController

### PR DESCRIPTION
**Pull request by librod89**
### What github issue is this PR for, if any?
#1590

### What changed, and why?
Fleshed out All-Casa Admin Request tests

### How will this affect user permissions?
Does not affect user permissions

### How is this tested? (please write tests!) 💖💪
N/A

### Screenshots please :)
Coverage was at 54.55%. Now it's at 100%.

image

### CODE REVIEW

File: spec/requests/all_casa_admins/casa_admins_spec.rb [hunk 1]
```diff
@@ -19,50 +27,101 @@
...
+        expect(response).to render_template "casa_admins/edit"
       end
+    end
+  end
 
+  describe "PATCH /deactivate" do
+    context "when current user is all casa admin" do
       it "should successfully deactivate another casa admin's profile" do
-        casa_admin.active = true
-        casa_admin.deactivate
```
Comment (librod89) [inline spec/requests/all_casa_admins/casa_admins_spec.rb:48]: `casa_admin.deactivate` and `casa_admin.activate` are both already being tested in spec/models/casa_admin_spec.rb so I changed these lines to access the controller methods.

**Comment by librod89** [comment]
I could not reproduce the cypress error locally.

**Review by compwron — APPROVED** [review]
Very nice tests!! :) :) 💪✨🙌🏼

**Comment by compwron** [comment]
The cypress failure looks like a n issue with db seed that I thought we had fixed :( I'll file another issue
> ActiveRecord::RecordInvalid: Validation failed: Volunteer must exist, Volunteer can't be blank, Volunteer Case assignee must be an active volunteer

**Pull request merged**